guides/source/active_record_migrations.md

```diff
@@ -539,6 +539,14 @@ definitions:
 `change_table` is also reversible, as long as the block does not call `change`,
 `change_default` or `remove`.
 
+`remove_column` is reversible if you supply the column type as a third
+argument. The column options should be provided too, so Rails will recreate the
+column exactly how it was before while rolling-back:
```

Comment:
How about this :smile:
"Provide the original column options too, otherwise Rails can't recreate the column exactly when rolling back."